src/transport/abstract-transport.js

```diff
@@ -20,6 +20,7 @@ function AbstractTransport(address, timeout, messageHandler) {
   this.awaitingPump = false;
   this.connected = false;
   this.backoffMilliseconds = 1;
+  this.nextReconnectListenerToken = 1;
```

Comment:
I'm not sure I fully understand the question.  This is the auto-incrementing (guaranteed unique) token returned from calls to `<some_transport>.addReconnectListener`.  It can be used by the caller to stop getting notified about reconnects with `<some_transport>.removeReconnectListener`.  This has nothing to do with blocks or block streams (though it is part of the foundation upon which reliable block streams are built).